net/ejabberd/Portfile

```diff
@@ -1,23 +1,23 @@
 PortSystem          1.0
 
 name                ejabberd
-version             18.12.1
-revision            2
+version             24.02
+revision            0
 categories          net
 platforms           darwin
```

Comment:
this is default setting for "platforms" so it can be removed

Reply:
@reneeotten Fixed it and reverted livecheck with correct URL.